README.md

```diff
@@ -212,6 +212,21 @@ extensions = ["com.romix.akka.serialization.kryo.KryoSerializationExtension$"]
             # class name to perform a custom initialization of Kryo instances in
             # addition to what is done automatically based on the config file.
             kryo-custom-serializer-init = "CustomKryoSerializerInitFQCN"
+			
+			# If enabled, allows Kryo to resolve subclasses of registered Types.
+			#
+			# This is primarily useful when idstrategy is set to "explicit". In this
+			# case, all classes to be serialized must be explicitly registered. The
+			# problem is that a large number of common Scala and Akka types (such as
+			# Map and ActorRef) are actually traits that mask a large number of
+			# specialized classes that deal with various situations and optimizations.
+			# It isn't straightforward to register all of these, so you can instead
+			# register a single supertype, with a serializer that can handle *all* of
+			# the subclasses, and the subclasses get serialized with that.
+			#
+			# Use this with care: you should only rely on this when you are confident
+			# that the superclass serializer covers all of the special cases properly. 
+			resolve-subclasses = false
 
```

Comment:
Please change the indentation of the comments block above to match the rest of the comments before and after it. It seems to be shifted to the right a bit too much.